hadoop-ozone/dev-support/checks/findbugs.sh

```diff
@@ -16,16 +16,15 @@
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 cd "$DIR/../../.." || exit 1
 
-mvn -B compile -fn findbugs:check -Dfindbugs.failOnError=false  -f pom.ozone.xml
+mvn -B compile -fn spotbugs:check -Dspotbugs.failOnError=false -f pom.ozone.xml
```

Comment:
should this be spotbugs:check or spotbugs:spotbugs ?? The spotbugs:check does not produce the report 

Please look at the table in this link.
https://spotbugs.github.io/spotbugs-maven-plugin/plugin-info.html

Just wondering if it makes sense to save the report at all .. ? this is more of a question, than informed opinion.

Reply:
Based on [this doc](https://spotbugs.readthedocs.io/en/latest/maven.html#goals-of-spotbugs-maven-plugin) I think `spotbugs:spotbugs` is the same as `spotbugs:check -Dspotbugs.failOnError=false`.